Maskbook/Scene/App/FIleService/ViewModel/FileServiceViewModel.swift

```diff
@@ -138,7 +138,7 @@ final class FileServiceViewModel: ObservableObject {
         self.actionSignal(.share(value))
     }
 
-    func tryUploading(_ fileItem: FileServiceSelectedFileItem) {
+    func tryUploading(_ fileItem: FileServiceSelectedFileItem, option: FileServiceUploadOption) {
```

Comment:
Move `option` as a property of `FileServiceSelectedFileItem ` ?

Reply:
I give you two model, user selected file and user selected option. before constructing as `FileServiceUploadingItem`, there are independent.